docs/operate/control.mdx

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 4
+description: "Control running servers and clusters"
+---
+
+import Admonition from '@theme/Admonition';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Control Servers
+
+Starting with version 1.2, Restate includes a command line utility to connect to and control running Restate servers called `restatectl`. This tool is specifically designed for system operators to manage the Restate servers, and is distinct from the application developer-focused `restate` CLI tool, which is used to manage service deployments and invocations.
+
+## Getting `restatectl`
+
+You can install `restatectl` using your preferred [installation method](/develop/local_dev). The tool is also included in the `restate-cli` and `restate` Docker images. To use `restatectl` in a running Docker container, you can use the following command from the host machine to connect to it without spawning a separate container:
+
+```shell
+docker exec &lt;CONTAINER_NAME&gt; restatectl [ARGUMENTS]
```

Comment:
This renders wrong in browser. I think it's safe to use `< >` signs inside a `shell` section

![image](https://github.com/user-attachments/assets/6f896081-0e77-4da9-a5a7-a2ff56ebd5c2)

Reply:
fixed, ty!